src/Service/DynamoDb/src/Result/TableNotExistsWaiter.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace AsyncAws\DynamoDb\Result;
+
+use AsyncAws\Core\Exception\Http\HttpException;
+use AsyncAws\Core\Response;
+use AsyncAws\Core\Waiter;
+use AsyncAws\DynamoDb\DynamoDbClient;
+use AsyncAws\DynamoDb\Input\DescribeTableInput;
+
+class TableNotExistsWaiter extends Waiter
+{
+    protected const WAIT_TIMEOUT = 500.0;
+    protected const WAIT_DELAY = 2.0;
```

Comment:
Whereas the default time is 20 seconds, the empty table are actually deleted in less that 2 seconds.